environment_gcc4.yml

```diff
@@ -1,12 +1,12 @@
 channels:
-  - conda-forge
+  - jessemapel
   - usgs-astrogeology
+  - conda-forge
+  - probcomp
   - defaults
 
 dependencies:
-  - libcxx
-  - cxx-compiler
-  - ale=0.6.4
+  - ale=0.7.2
```

Comment:
Since I have to push a custom ALE build, I'm leaving this hard pinned to this version. Allowing this to be greater, would pull a higher version off conda-forge if available.